manifests/params.pp

```diff
@@ -6,7 +6,6 @@
 class consul_template::params {
 
   $install_method     = 'url'
-  $log_level          = 'info'
```

Comment:
This is a consul_template configuration option and doesn't really belong with all of the puppet module params so it's been (re)moved.